src/openrct2/world/Location.hpp

```diff
@@ -149,27 +149,38 @@ struct CoordsXY
     {
         return { floor2(x, 32), floor2(y, 32) };
     }
+
+    bool isNull() const
+    {
+        return x == COORDS_NULL;
+    };
```

Comment:
While we're at it, could you also define a setNull() function?

Reply:
I'm not sure I like that, semantically. Is that a common use-case? Perhaps re-assigning from a singleton would be more appropriate?

Reply:
Unsure if I create the `setNull()` or not

Reply:
@AaronVanGeffen Code that uses Coords should not have too intimate knowledge of how to mark coordinates as null. That's why it needs to be a function. Adding a `setNull()` function is the most straightforward way to do so.